infra/terraform/gcp/staging/provider.tf

```diff
@@ -0,0 +1,5 @@
+provider "google" {
+  credentials = file("${path.module}/../assets/airqo-terraform.json")
```

Comment:
I am getting a runtime error when I run `terraform plan`, something about "path.module" not being present. Any hints as I also continue to look into it? @danielogen 

Detailed error message is shared below:
```
Error: Invalid function argument

  on provider.tf line 2, in provider "google":
   2:   credentials = file("${path.module}/../assets/airqo-terraform.json")
    |----------------
    | path.module is "."

Invalid value for "path" parameter: no file exists at
../assets/airqo-terraform.json; this function works only with files that are
distributed as part of the configuration source code, so if this file will be
created by a resource in this configuration you must instead obtain this
result from an attribute of that resource.
```

Reply:
All is good now, thanks.